kubernetes/loculus/templates/_common-metadata.tpl

```diff
@@ -175,6 +175,9 @@ organisms:
       {{- with ($instance.schema | include "loculus.patchMetadataSchema" | fromYaml) }}
       organismName: {{ quote .organismName }}
       loadSequencesAutomatically: {{ .loadSequencesAutomatically | default false }}
+      {{ if .richFastaHeaderFields}}
+      richFastaHeaderFields: {{ toJson .richFastaHeaderFields }}
+      {{ end }}
```

Comment:
How should we treat an empty array? Should we treat it as equivalent to not having set it at all?

Reply:
Sounds good as well! Can we define that in the schema?

Reply:
The JSON schema has an option `"minItems": 1` that let's us do that.